src/PFire.Core/Protocol/XFireAttributes/UInt32Attribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
```

Comment:
If you can please also delete the `Int32Attribute` class in this PR too

Reply:
Will get on it.